storybookjs/storybook
# Return empty array when Array knob is empty

Issue: #1404

## What I did
Made the array knob return an empty array instead of `[""]` when the input textbox is empty. Also added a test case.

## How to test
Run the new test case.

Is this testable with jest or storyshots? jest

Does this need a new example in the kitchen sink apps? nah

Does this need an update to the documentation? probably not...

If your answer is yes to any of these, please make sure to include it in your PR.
